requirements/base.txt

```diff
@@ -4,7 +4,7 @@ django-basicauth==0.5.3
 django-extensions==3.1.3
 django-localflavor==3.0.1
 django-markdown-deux==1.0.5
-django==2.2.22  # pyup: >=2.2,<3.0
+Django<=3.3 # pyup: >=2.2,<3.2
```

Comment:
Drive by, but for an application (rather than a library) this should be a pin not a range. You don't want a surprise django upgrade (or downgrade - `<=3.3` really gives you some scope :laughing: ) when you deploy :grimacing:

Also pyup hasn't been running on this repo for over a year. Pyup comments can be removed. If you want to direct dependabot to ignore upgrades to the next LTS, the syntax is https://github.com/DemocracyClub/WhoCanIVoteFor/blob/efabcb7078d1f9be05baba746475ca11deb49e44/.github/dependabot.yml#L8-L10 although it would want to be `versions: [">=4.0", "<4.2"]`. In fact, that probably wants bumping on the WCVIF repo too. It looks like you've upgraded to 3.2 over there and overlooked it.